powersimdata/scenario/create.py

```diff
@@ -432,19 +435,36 @@ def __init__(self, grid_model, interconnect, table, **kwargs):
         super().__init__(grid_model, interconnect, table)
 
         self.reduction = None if "reduction" not in kwargs else kwargs["reduction"]
+        self.zenodo_record_id = (
+            None if "zenodo_record_id" not in kwargs else kwargs["zenodo_record_id"]
+        )
 
         self.print_existing_study()
 
         self.set_base_grid()
+        self.set_base_profile()
         self.set_change_table()
 
     def set_base_grid(self):
         """Set base grid"""
-        raise NotImplementedError()
+        self.base_grid = Grid(
+            self.interconnect,
+            source=self.grid_model,
+            reduction=self.reduction,
+            zenodo_record_id=self.zenodo_record_id,
+        )
+
+    def set_base_profile(self):
+        """Set base profile."""
+        version = f"v_{self.base_grid.version}"
```

Comment:
Wonder if we should keep the `v_` prefix since we already have `grid.version` set to e.g. `v0.6.1`

Reply:
Done